warehouse/static/js/warehouse/index.js

```diff
@@ -28,6 +28,15 @@ import * as formUtils from "warehouse/utils/forms";
 import Clipboard from "clipboard";
 import PositionWarning from "warehouse/utils/position-warning";
 import Statuspage from "warehouse/utils/statuspage";
+import projectTabs from "warehouse/utils/project-tabs";
+
+// project detail tabs
+docReady(()=> {
+  var headingBtn = document.querySelector(".-js-vertical-tab-content");
+  var mobileBtn = document.querySelector(".-js-vertical-tab-mobile-heading");
+  var styleProps = getComputedStyle(mobileBtn, null);
```

Comment:
mm, I think `let` is more for wrangling with scope but `const` would make more sense since they are not being reassigned anywhere in this scope.